src/get-started/install.md

```diff
@@ -44,9 +44,22 @@ For distributives with `deb`-based package managers (such as Debian or Ubuntu),
 use
 
 ```shell
-apt-get install libssl-dev pkg-config
+apt-get install libsodium-dev libsnappy-dev \
+    librocksdb-dev libssl-dev pkg-config
 ```
 
+`libsodium-dev`, `libsnappy-dev` and `librocksdb-dev` are optional libraries
+for installation. Meanwhile, as these packages reduce the build time, thus,
+their installation is highly recommended.
+
+You should also indicate the routes to these packages in the environment
+variables:
```

Comment:
You should also set the paths to libraries of these packages in the environment variables: